rdr_service/model/bq_questionnaires.py

```diff
@@ -133,9 +132,12 @@ def get_fields(self):
                             continue
 
                         if bq_field_name.lower() not in skip_fieldnames_lower:
+                            # If field is in force boolean list, set to integer.
+                            fld_type = BQFieldTypeEnum.INTEGER.name if bq_field_name.lower() in force_boolean_lower \
+                                        else BQFieldTypeEnum.STRING.name
```

Comment:
I'm trying to understand the `force_boolean_lower` logic. Is this casting all the values for specified codes to integers? What would it do for dates in these answers?

Reply:
All of the question codes/ fields that are in the `force_boolean_fields` list for the impacted modules are free text fields.   The idea is to just confirm if the free text field has any content or not, and set it to 1 if it does.

Reply:
If it would help to talk offline about the history of why I used lowercase string comparisons in the existing code, let me know. But it had to do with "not production quality" code table data in our lower environments, where we could have two question codes with nearly identical value strings except for matching on case or because one had a space where the other had an underscore.  It caused some unexpected "collisions" on results from `self.make_bq_field_name` when trying to generate a BigQuery table schema/column list.   

I don't know if converting the `_force_boolean_fields` values to lowercase is needed here but I don't think it has any negative side effects when testing for inclusion.  We're still assigning the unchanged `bq_field_name` value on line 139.

Reply:
It's been long enough I can't remember why I used the "lower()" when checking code values, but it doesn't hurt to have this extra step to ensure the equality check is successful.